projects/components/src/table/table.component.ts

```diff
@@ -158,7 +158,7 @@ import { TableColumnConfigExtended, TableService } from './table.service';
             selectable: this.supportsRowSelection()
           }"
           class="data-row"
-          [style.height]="this.rowHeight"
+          [style.min-height]="this.rowHeight"
```

Comment:
We should be ok with using min-height here as long as we verify all the existing tables and override any broken places with explicit height.

We reverted this change last time (I was the author) since we didn't consider individual height overrides then @jake-bassett